### PR TITLE
feat(core): 11953 improve development experience for vite linking to dn

### PR DESCRIPTION
**Pull request by Akiyamka**
image

### CODE REVIEW

File: scripts/auto-link/config.mjs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { readFileSync, writeFileSync } from "fs";
+import prompts from './prompts.mjs';
+import { logger } from './logger.mjs';
+
+const log = logger('Config reader')
+// @ts-check
```
Comment (IlyaIzr) [inline scripts/auto-link/config.mjs:6]:
```suggestion

```

**Comment by IlyaIzr** [comment]
[Winows 11]
So i've tried absolute path to ui kit (with win slashes and backslashes also)
We could improve error message
Also despite path being unreadable the path got saved so next time i run script - it just throws error again (i'd suggest not saving it if path is not OK)
image
image
relative path `../ui` also didn't work for me

Tried both on powershell and bash

File: scripts/auto-link/config.mjs (hunk 1 above)
Comment (Akiyamka) [reply scripts/auto-link/config.mjs:6]: Why you suggest delete this ?

Comment (IlyaIzr) [reply scripts/auto-link/config.mjs:6]: Isn't it checked by ts anyway?

Comment (Akiyamka) [reply scripts/auto-link/config.mjs:6]: it checked in vscode

**Pull request merged**